std_msgs/Header header

video_analyzer/VideoAnalysisObject[] objects
